charts/posthog/README.md

```diff
@@ -100,24 +182,20 @@ Dependent charts can also have values overwritten. Preface values with postgresq
 
 All configuration options can be found in [ALL_VALUES.md](ALL_VALUES.md) or in [values.yaml](values.yaml) file.
 
-### Example `values.yaml` (<1M events/month)
+
+### Scaling up
+The default configuration is geared towards minimizing costs. Here are example extra values overrides to use for scaling up:
 <details>
   <summary>
-    <b>Click to view</b>
+    <b> Additional values to `values.yaml` for <1M events/month</b>
   </summary>
 
 <br />
 
-This `values.yaml` is geared towards minimizing cost on google kubernetes engine. Use it if you have less than 1M events/month.
-
 ```yaml
-# Minimal settings for scale of 10k to 1M events/month.
 # Note that this is experimental, please let us know how this worked for you.
 
-cloud: gcp
```

Comment:
Why? Without this this yaml file does not work.

Reply:
This is gcp specific, I moved that part up to the minimal required configuration within install section, the sizing stuff is otherwise is platform agnostic.